client/buildSrc/src/main/java/Dependencies.kt

```diff
@@ -1,100 +1,92 @@
+private const val AGP_VERSION = "7.0.0-alpha08"
+private const val ANDROIDX_HILT_VERSION = "1.0.0-alpha03"
+private const val DAGGER_HILT_VERSION = "2.32-alpha"
+private const val KOTLIN_VERSION = "1.4.30"
+private const val LINT_VERSION = "27.3.0-alpha04"
+
+object Plugins {
+
+    const val agp = "com.android.tools.build:gradle:$AGP_VERSION"
+    const val agpBuilder = "com.android.tools.build:builder:$AGP_VERSION"
+    const val agpBuilderModel = "com.android.tools.build:builder-model:$AGP_VERSION"
+    const val agpLintModel = "com.android.tools.lint:lint-model:$LINT_VERSION"
+    const val kotlin = "org.jetbrains.kotlin:kotlin-gradle-plugin:$KOTLIN_VERSION"
+    const val hilt = "com.google.dagger:hilt-android-gradle-plugin:$DAGGER_HILT_VERSION"
+}
+
 object Dependencies {
 
     object Kotlin {
 
-        private const val kotlin_version = "1.4.1"
-
-        val kotlin = "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${kotlin_version}"
-
         object Coroutines {
+            private const val version = "1.4.1"
 
-            private const val coroutines_version = "1.3.9"
-
-            const val coroutines =
-                "org.jetbrains.kotlinx:kotlinx-coroutines-android:${coroutines_version}"
-            const val core =
-                "org.jetbrains.kotlinx:kotlinx-coroutines-core:${coroutines_version}"
+            const val coroutines = "org.jetbrains.kotlinx:kotlinx-coroutines-android:$version"
+            const val core = "org.jetbrains.kotlinx:kotlinx-coroutines-core:$version"
         }
-
     }
 
     object AndroidX {
 
-        private const val appcompat_version = "1.2.0"
-        private const val constraint_layout_version = "2.0.4"
-        private const val legacy_support_version = "1.0.0"
-        private const val work_runtime_version = "2.5.0"
-        private const val multidex_version = "2.0.1"
-
-        const val appcompat = "androidx.appcompat:appcompat:$appcompat_version"
-        const val constraintlayout = "androidx.constraintlayout:constraintlayout:$constraint_layout_version"
-        const val legacy_support = "androidx.legacy:legacy-support-v4:$legacy_support_version"
-        const val work_runtime = "androidx.work:work-runtime-ktx:$work_runtime_version"
-        const val multidex = "androidx.multidex:multidex:$multidex_version"
-
+        const val appcompat = "androidx.appcompat:appcompat:1.3.0-beta01"
```

Comment:
Should we separate version number from library name?

Reply:
Don't really think that it has any benefit if the version number is only used at one place.

Reply:
But it might offer a easy method to change version numbers but I do get the point